_includes/footer.html

```diff
@@ -49,11 +49,13 @@
                   <div class="headline"><h2>{% t footer-contact.title  %}</h2></div>                         <address class="md-margin-bottom-40">
                     <a href="https://www.google.co.uk/maps/place/Codurance/@51.5238156,-0.1025175,17z/data=!4m13!1m7!3m6!1s0x48761b50d18bf817:0xcce7f698e8aeba03!2s15+Northburgh+St,+London+EC1V+0JR!3b1!8m2!3d51.5238156!4d-0.1003288!3m4!1s0x48761b50c8c7114f:0x6245fac5ecb52d7f!8m2!3d51.5238487!4d-0.1004723" target="_blank">15 Northburgh Street <br> London EC1V 0JR</a>
                     <br>
+                    {% t footer-contact.phone_title %}: +44 207 4902967
+                    <br>
                     <br>
                     <a href="https://www.google.co.uk/maps/place/Carrer+d'Arag%C3%B3,+208,+08011+Barcelona,+Spain/@41.3876335,2.1571825,17z/data=!3m1!4b1!4m5!3m4!1s0x12a4a28e9a736993:0x13c80b7ea4d9fb0f!8m2!3d41.3876335!4d2.1593712" target="_blank">Carrer Aragó, 208 <br> 08011, Barcelona</a>
                     <br>
+                    {% t footer-contact.phone_title %}: +34 689 723 737
```

Comment:
would it be possible to use variables here as well ?

Reply:
I think it's not possible with the current situation, because I would like to indicate both telephone numbers (London and Barcelona) at the same time and I think variables work together the language. I'll review it if I have some idea to use variables in this case.

Reply:
Thanks @cyplo !!